### ISSUE TITLE
Version 5.5.2 gives exception 'An item with the same key has already been added. Key'

### ISSUE THREAD

**Issue by charneca** [EPPlusSoftware/EPPlus#284]
Hello.

When opening an Excel template,  adding data and finally saving,  it gives the exception **"ArgumentException: An item with the same key has already been added. Key:  "**

On version 5.0.4 it saves the changes without any exception.

I have attached a sample Excel File.
[test.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/5850620/test.xlsx)

I am using .net 5 and here it is some code to generate the error:
```
        public class Test 
        {
            public int Value1 { get; set; }
            public int Value2 { get; set; }
            public int Value3 { get; set; }

        }

        /// <summary>
        ///  debug purposes
        /// </summary>
        /// <returns></returns>
        public string TestExcel()
        {
            //1
            var report1 = new List<Test>
            { 
                new Test{ Value1 = 1, Value2= 2, Value3=3 },
                new Test{ Value1 = 2, Value2= 3, Value3=4 },
                new Test{ Value1 = 5, Value2= 6, Value3=7 }
            };

            //3
            var report2 = new List<Test>
            {
                new Test{ Value1 = 0, Value2= 0, Value3=0 },
                new Test{ Value1 = 0, Value2= 0, Value3=0 },
                new Test{ Value1 = 0, Value2= 0, Value3=0 }
            };

            //4
            var report3 = new List<Test>
            {
                new Test{ Value1 = 3, Value2= 3, Value3=3 },
                new Test{ Value1 = 3, Value2= 3, Value3=3 },
                new Test{ Value1 = 3, Value2= 3, Value3=3 }
            };


            string workingDirectory = AppDomain.CurrentDomain.BaseDirectory;
            FileInfo template = new(workingDirectory + @"Templates\test.xlsx");
            FileInfo finalFile = new(string.Format(workingDirectory + @"Cache\test_{0}.xlsx", DateTime.Now.Ticks));

            finalFile.Delete();

            using var excelFile = new ExcelPackage(template);

            //Data1
            var worksheet = excelFile.Workbook.Worksheets["Test1"];
            ExcelRangeBase location = worksheet.Cells["A1"].LoadFromCollection(Collection: report1, PrintHeaders: true);
            worksheet.Tables.Add(location, "mytestTbl");

            //Data2
            worksheet = excelFile.Workbook.Worksheets["Test2"];
            location = worksheet.Cells["A1"].LoadFromCollection(Collection: report2, PrintHeaders: true);
            worksheet.Tables.Add(location, "mytestsureTbl");

            //Data3
            location = worksheet.Cells["K1"].LoadFromCollection(Collection: report3, PrintHeaders: true);
            worksheet.Tables.Add(location, "Test3");



            var wsFirst = excelFile.Workbook.Worksheets["Test1"];

            wsFirst.Select();


            excelFile.SaveAs(finalFile);


            return finalFile.FullName;
        }
```

**Comment by JanKallman** [comment]
This happends when a pivot cache have both a null value (\<m/>) and an empty string. I have made a fix with a redesign of the shared items handling. Please verify that it works for you using our develop nuget feed https://ci.appveyor.com/nuget/epplus (add it to package sources inside Visual Studio).

**Comment by charneca** [comment]
> This happends when a pivot cache have both a null value (<m/>) and an empty string. I have made a fix with a redesign of the shared items handling. Please verify that it works for you using our develop nuget feed https://ci.appveyor.com/nuget/epplus (add it to package sources inside Visual Studio).


Hello, 

yes this seems to solve this issue, thank you.
I have another issue with another template, but I will create another post.

**Comment by JanKallman** [comment]
Fixed in 5.5.3
